src/Resources/views/bundles/SyliusShopBundle/Common/Order/_addresses.html.twig

```diff
@@ -0,0 +1,23 @@
+<div class="ui segment">
+    <div class="ui {% if order.isShippingRequired() %}two{% else %}one{% endif %} column divided stackable grid">
```

Comment:
maybe if we added block from line https://github.com/Sylius/PayPalPlugin/pull/153/files#diff-1d7e4473bdf0d61d613efa015fe5c9998422a98086d812a6a7868cff88a2e6d0R15 it would look better to display info about missing address instead of comma.
![image](https://user-images.githubusercontent.com/22825722/98273769-e86bd780-1f92-11eb-87f8-d05ad26e99ca.png)

What do you think?

Reply:
Agreed 👍 However, it would require from us extending another template: `@SyliusShop/Common/_address.html.twig`, which will be also another step of the plugin's installation. I would vote to fix it in Sylius itself (not displaying the comma if there is no street in the address) 🖖